doc/api/worker_threads.md

```diff
@@ -913,6 +913,26 @@ Calling `unref()` on a worker will allow the thread to exit if this is the only
 active handle in the event system. If the worker is already `unref()`ed calling
 `unref()` again will have no effect.
 
+### `worker.performance.eventLoopUtilization([utilization1[, utilization2]])`
```

Comment:
Entries are in alphabetical order. So I want to suggest moving this below `worker.getHeapSnapshot()` and above `worker.postMessage()`, but....

Do we need to add an entry (even if it's just a single sentence) for `worker.performance` and make this a subheader of that entry? If so, where would that go in correct order? @nodejs/documentation

Reply:
@Trott Eventually I'd like the `worker.performance` object to contain the `nodeTiming` object. If we don't think that will ever happen then I'd suggest we change the API to `worker.eventLoopUtilization()`.